docs/apm/troubleshooting.asciidoc

```diff
@@ -69,3 +69,31 @@ or because something is happening to the request that the Agent doesn't understa
 To resolve this, you'll need to head over to the relevant {apm-agents-ref}[Agent documentation].
 Specifically, view the Agent's supported technologies page.
 You can also use the Agent's public API to manually set a name for the transaction.
+
+==== Fields are not indexed or searchable
+
+In Elasticsearch, index templates are used to define settings and mappings that determine how fields should be analyzed.
+The recommended index template file for APM Server is installed by the APM Server packages.
+This template, by default, enables and disables indexing on certain fields.
```

Comment:
The fields are still indexed, but the template makes APM fields visible in all of Kibana for features like the kuery bar or linking to other plugins like Logs, Uptime, and Discover.

Reply:
Again, I think this is mixing Kibana index patterns and index templates. Afair it's possible to disable indexing of specific fields via the index template (`enabled: false` for a specific field).